README.md

```diff
@@ -1,53 +1,83 @@
-# Role Name
+# mssql
 ![CI Testing](https://github.com/linux-system-roles/template/workflows/tox/badge.svg)
 
-A template for an ansible role which configures some GNU/Linux subsystem or
-service. A brief description of the role goes here.
+This role installs and configures mssql on Red Hat Enterprise Linux 8.
```

Comment:
We should be consistent with the naming.  I suggest here, for the first mention of the product, we use "This role installs and configures Microsoft SQL Server (MSSQL) ..." then later in the doc use "MSSQL" when referring to the product name, and "mssql" when referring to the role name.

Reply:
Is RHEL7 supported also?  It's in the meta/main.yml file.